packages/default-storage/ios/RNCAsyncStorage.mm

```diff
@@ -869,8 +869,14 @@ - (BOOL)_passthroughDelegate
 
     [_manifest removeAllObjects];
     [RCTGetCache() removeAllObjects];
-    NSDictionary *error = RCTDeleteStorageDirectory();
```

Comment:
I think it would be simpler if we check whether `error.code == NSFileNoSuchFileError` in `RCTDeleteStorageDirectory()` and ignore it.

Reply:
yeah, it works and is also more clean, thanks for that.
I'm kinda new on Objective-C, I only return nil to ignore, if it was better to return another thing I could change again.